src-tauri/src/main.rs

```diff
@@ -145,6 +146,7 @@ impl Default for AppSettings {
             editor_preview_resolution: Some(1920),
             sort_criteria: None,
             theme: Some("dark".to_string()),
+            controls: Some("app".to_string()),
```

Comment:
Oh, also, I left the RapidRAW custom styled controls as the default setting here.

Just change this to `"native"` instead of `"app"` if you think it should default to the native controls. Personally, I think the custom styled ones look pretty nice anyway, but I think there should be the option. In my case, with my usual window manager, my application windows all have very thin titlebars with only the title text, and there are no control buttons at all, so native mode fits in better. But I know that setups like mine are by far a tiny minority.